### ISSUE TITLE
URL Image List yControl is not displaying the images.

### ISSUE THREAD

**Issue by cyus** [DynamicsNinja/PCF-Url-Image-List-Control#2]
Hi Ivan, It worked for the read only grid but didn't not for the URL Image control, same with the PCF Carousel Control, the image is not showing as well. Any idea? Thank you

![image](https://user-images.githubusercontent.com/28690471/79827942-38f25d80-83d2-11ea-9879-34fcdae317fb.png)

**Comment by DynamicsNinja** [comment]
Did you configure the control to show in the web interface? Can you send me a screenshot of the control configuration so I can see what is wrong there?

**Comment by cyus** [comment]
Hi here it is,
![image](https://user-images.githubusercontent.com/28690471/79875408-33ba0080-841c-11ea-9044-4d1ad37463c1.png)

**Comment by DynamicsNinja** [comment]
Everything seems fine in the config. Did you do it on the view or entity level?

**Comment by cyus** [comment]
I added it on the view & the code basically cloned. May be something wrong here in my web control configuration?
![image](https://user-images.githubusercontent.com/28690471/79924865-8e2e7d80-846b-11ea-8e89-b3e7932a2876.png)

**Comment by DynamicsNinja** [comment]
Do you have something like this in your view?

![Image 557](https://user-images.githubusercontent.com/31212465/79975154-d31dd880-849a-11ea-8580-8705bbff0c07.png)

**Comment by cyus** [comment]
Yes I do Let me try this again later by forking, is the images above images of the data set cars you have in the environment?

**Comment by DynamicsNinja** [comment]
Yes, that's the example from one of my environments. Cool, let me know how did it go.
